CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ if (POLICY CMP0042)
   cmake_policy (SET CMP0042 NEW)
 endif()
 
-option ( LINK_PYTHON_LIBRARY "Link against python libraries" ON )
+if ( APPLE )
+  option ( LINK_PYTHON_LIBRARY "Link against python libraries - NOT RECOMMENDED" OFF )
+else ()
+  option ( LINK_PYTHON_LIBRARY "Link against python libraries" ON )
+endif ()
```

Comment:
Is that change still required, when `-undefined dynamic_lookup` is added to link flags?

Reply:
I think so, since the recommendations are: "Please use -undefined dynamic_lookup *instead of* -lpython or -framework Python"

Reply:
the LINK_PYTHON_LIBRARY option can be removed, then linking is only needed on windows, so the code would look like this:

if (WIN32)
  swig_link_libraries (${MODULENAME}_python ${PYTHON_LIBRARIES})
elseif(APPLE)
  set_target_properties(${MODULENAME}_python PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
endif ()

Reply:
Oh, right. As one no longer links with libpython, there are undefined symbol, and that is why the `-undefined` flags is required. OK, thanks.